examples/circuitplayground_touch_all.py

```diff
@@ -2,20 +2,27 @@
 # SPDX-License-Identifier: MIT
 
 """This example prints to the serial console when you touch the capacitive touch pads."""
+import board
 from adafruit_circuitplayground import cp
 
+
+# You'll need to first use the touchpads individually to register them as active touchpads
+# You don't have to use the result though
+is_a1_touched = cp.touch_A1  # This result can be used if you want
+if is_a1_touched:
+    print("A1 was touched upon startup!")
+is_a2_touched = cp.touch_A2
+is_a3_touched = cp.touch_A3
+is_a4_touched = cp.touch_A4
+
+
 while True:
-    if cp.touch_A1:
-        print("Touched pad A1")
-    if cp.touch_A2:
-        print("Touched pad A2")
-    if cp.touch_A3:
-        print("Touched pad A3")
-    if cp.touch_A4:
-        print("Touched pad A4")
-    if cp.touch_A5:
-        print("Touched pad A5")
-    if cp.touch_A6:
-        print("Touched pad A6")
-    if cp.touch_TX:
-        print("Touched pad TX")
+
+    print("Pads that are currently setup as touchpads:")
+    print(cp.touch_pins)
+
+    print("Touchpads currently registering a touch:")
+    print(cp.touched)
```

Comment:
I would recommend wrapping these prints in an if statement so they only occur when there are actually touches rather than printing the empty list every iteration if there aren't any pins being touched.

e.g.
```
    cur_touched = cp.touched
    if cur_touched:
        print("Touchpads currently registering a touch:")
        print(cur_touched)
```